### DISCUSSION TITLE
Sonarr Daily Episode Trimmer

### DISCUSSION THREAD

**Discussion by Icenova20** [RandomNinjaAtk/arr-scripts#282] [Q&A] [answered]
Is there a way to have this script unmonitor episodes it has trimmed? Or maybe it already does that and i have just made a mis-configuration.

I'm seeing that often super old daily episodes will download only to be immediately deleted as they are not recent enough to be one of the 14 that are kept.

**Comment by jaycedk** [comment]
Try this, it might work.
![image](https://github.com/RandomNinjaAtk/arr-scripts/assets/17232146/6f94fb1e-ebda-42c2-9a35-727225241353)

**Reply by Icenova20** [reply]
I appreciate the reply.

I used to have that setting checked and while it did work, I found that it only allowed episodes to upgrade once, because after an episode would upgrade it would then unmonitor the episode and never upgrade again.

**Reply by jaycedk** [reply]
Ok that all I can think of.
You could try again, that bug you describe might have been squashed 🪲

**Comment by RandomNinjaAtk** [comment, marked as answer]
The script should be unmonitoring them.

**Reply by Icenova20** [reply]
okay i can confirm it does.

i guess i just manually monitored old stuff at some point on accident.
